recipes/deepspeed/meta.yaml

```
@@ -0,0 +1,96 @@
+{% set name = "deepspeed" %}
+{% set version = "0.7.7" %}
+
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5069fefabbd61dcff64e322189310908ac0291bbda0d472fe89de1cd82d417af
+
+build:
+  number: 0
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                                # [cuda_compiler_version == "None"]
+  skip: true  # [win or cuda_compiler_version in ("10.2",)]
```

```suggestion
  skip: true  # [win]
  skip: true  # [cuda_compiler_version in ("10.2", "11.0", "11.1")]
```

We stopped shipping newer pytorch all but 11.2

---

Thanks.